src/inverseRobot.js

```diff
@@ -23,8 +23,24 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = i + 1; j < values.length; j++) {
+      if (values[j] === values[i]) {
+        return null;
+      }
+    }
+  }
+
+  const swapped = Object.fromEntries(
+    // преобразовать в массив, затем map, затем fromEntries обратно объект
+    Object.entries(robot).map(([key, value]) => [value, key])
+  );
+
+  return swapped;
```

Comment:
So many loops and iterations 😟 
You can just
* create new object (`newRobot` for example )
* iterate through keys (use `for ... in`)
* in loop check if newRobot has property `robot[key]` -> return null
* in another case add new property to newRobot

Reply:
Thank you :)